Description: mpd.service: Set the Documentation option
 This makes references to mpd(1) and mpd(5) appear in systemd status output.
Author: Wieland Hoffmann <themineo@gmail.com>
Origin: upstream

diff --git a/systemd/mpd.service.in b/systemd/mpd.service.in
index a72eb92..caf0d67 100644
--- a/systemd/mpd.service.in
+++ b/systemd/mpd.service.in
@@ -1,5 +1,6 @@
 [Unit]
 Description=Music Player Daemon
+Documentation=man:mpd(1) man:mpd.conf(5)
 After=network.target sound.target
 
 [Service]
